libtelemetry/src/main/java/com/mapbox/android/telemetry/VisionAttachment.java

```diff
@@ -0,0 +1,23 @@
+package com.mapbox.android.telemetry;
+
+import okhttp3.MediaType;
+
+public class VisionAttachment {
```

Comment:
Should name of this class be more generic? One suggestion could be: FileAttachment. Thoughts?